jumanji/wrappers.py

```diff
@@ -215,6 +215,22 @@ def action_spec(self) -> dm_env.specs.Array:
     def unwrapped(self) -> Environment:
         return self._env
 
+    @property
+    def state(self) -> Any:
+        return self._state
+
+    @state.setter
+    def state(self, new_state: Any) -> None:
+        self._state = new_state
+
+    @property
+    def key(self) -> PRNGKey:
+        return self._key
+
+    @key.setter
+    def key(self, new_key: PRNGKey) -> None:
+        self._key = new_key
```

Comment:
It is not what has been asked in the issue https://github.com/instadeepai/jumanji/issues/27

Reply:
I can but it is not what has been asked in [the issue](https://github.com/instadeepai/jumanji/issues/27). Sorry I forgot to link the pr to the issue

Reply:
Sorry, I have discussed this with @clement-bonnet and we agreed to expose these as public attributes because that is a more compact/cleaner solution.

Thank you for your interest and I am looking forward to reviewing your final contribution. 🙂